### DISCUSSION TITLE
sphinx 7.3 incompatible

### DISCUSSION THREAD

**Discussion by gaborbernat** [pradyunsg/furo#782] [Potential Issue] [answered]
The 'furo' theme inherits from 'basic-ng', which is not a loaded theme. Loaded themes are: agogo, alabaster, basic, bizstyle, classic, default, epub, furo, haiku, nature, nonav, pyramid, scrolls, sphinxdoc, traditional

**Comment by AA-Turner** [comment, marked as answer]
See: https://github.com/sphinx-doc/sphinx/issues/12292. This is fixed in Sphinx 7.3.2.

A

**Reply by pradyunsg** [reply]
And the corresponding PR in Furo has been merged and released as well: https://github.com/pradyunsg/furo/pull/783

Thanks @AA-Turner! ^.^
